datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -113,6 +126,13 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
             "Neither EXTRA_QUESTIONNAIRE_JSON_URI nor EXTRA_QUESTIONNAIRE_JSON_STRING is supplied.",
           )
       }
+    // Add timestamp questionnaire items
+    getTimestampItem("launch-timestamp")?.let {
+        questionnaire.addItem(it)
+    }
+    getTimestampItem("submission-timestamp")?.let {
+        questionnaire.addItem(it)
+    }
```

Comment:
```suggestion
if (shouldAddTimestampItems) {
questionnaire.addItem(getTimestampItem("launch-timestamp"))
questionnaire.addItem(getTimestampItem("submission-timestamp"))
}
```

It is easier to read with the if condition here than to figure out from exploring the `getTimestampItem` function firsts to know what's going on.

Or 
maybe rename `getTimestampItem` to something that explains both its intents better.